dotnet-desktop-guide/framework/wpf/controls/snippets/how-to-add-a-watermark-to-a-textbox/csharp/TextBoxHelper.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <Namespaces>
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Media;
+// </Namespaces>
+
+namespace DotnetDocsSample
+{
```

Comment:
I'd switch to a file-based namespace. (If you use Visual Studio, it will automatically reformat the file for you.)

```suggestion
namespace DotnetDocsSample;
```

Reply:
.NET Framework :)